Reversed the Rsnow from 10000 to 1000 as consistent with AQMv7

KaiWang-NOAA opened the pull request:
Update ASX_DATA_MOD.F to reverse Rsnow to 1000 which is consistent with AQMv7 and CMAQv5.2.1. 

<!--Instructions: All subsequent sections of text should be filled in as appropriate.-->
## PR Checklist
- This PR has been tested on an RDHPCS machine and/or WCOSS2. Please select below:
  - [x] RDHPCS.
  - [x] WCOSS2.

- This PR has been tested using:
  - [x] The ufs-srweather-app workflow.
  - [ ] The AQM-Workflow workflow

- [ ] New or updated input data is required by this PR. <!-- If checked, please work with the code managers to update input data sets on all platforms.-->

- [ ] Baselines are expected to change.

## Description
<!--Provide a detailed description of what this PR does.-->
This PR will address the overprediction of O3 over snow-covered regions in AQMv8/CMAQv5.4

## Issue(s) addressed
<!--Please link the issues to be closed with this PR.
(Remember, issues must always be created before starting work on a PR branch!)
EX: - fixes #<issue_number>-->
- closes NOAA-EMC/AQM#113

## Dependencies
<!--If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).
Do PRs in upstream/other repositories need to be merged along with this one?
EX: - waiting on noaa-emc/fv3atm/pull/<pr_number>-->

KaiWang-NOAA committed:
Reversed the Rsnow from 10000 to 1000 as consistent with AQMv7

Update ASX_DATA_MOD.F to reverse Rsnow to 1000 which is consistent with AQMv7 and CMAQv5.2.1

BrianCurtis-NOAA commented:
@KaiWang-NOAA You haven't checked off that baselines are changing. Do they change with the change of the Rsnow value?

BrianCurtis-NOAA commented:
@KaiWang-NOAA Please run this using the AQM-Workflow as well. (the test version that you've had success with is fine for now)

BrianCurtis-NOAA committed:
Update pull_request_template.md

We need to make sure this is tested on at least the AQM-Workflow but for now the srw-app workflow is OK.

bbakernoaa committed:
Merge branch 'develop' into CMAQv5.4_fixedRsnow

bbakernoaa commented:
@KaiWang-NOAA what is the status of this?

bbakernoaa committed:
Merge branch 'develop' into CMAQv5.4_fixedRsnow

KaiWang-NOAA commented:
@Barry Baker - NOAA Federal ***@***.***> I'm combining this
commit with my recently updated AQM code. So I'll include this change with
a new PR and close this PR accordingly.

On Thu, Aug 7, 2025 at 12:52 PM Barry Baker ***@***.***>
wrote:

> *bbakernoaa* left a comment (NOAA-EMC/AQM#114)
> <https://github.com/NOAA-EMC/AQM/pull/114#issuecomment-3165010017>
>
> @KaiWang-NOAA <https://github.com/KaiWang-NOAA> what is the status of
> this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/AQM/pull/114#issuecomment-3165010017>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2A5ALSYZH3G3DE73ABHUSL3MN74FAVCNFSM6AAAAAB7DHRSUGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTCNRVGAYTAMBRG4>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>


-- 

Kai Wang, Ph.D.

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct., Rm. 2029

College Park, MD 20740

***@***.***

(301) 683-3230

KaiWang-NOAA committed:
Revert "Merge branch 'develop' into CMAQv5.4_fixedRsnow"

This reverts commit d8808edc4c5fcb2a41d79ae9852395123dea2556, reversing
changes made to 9ed2a9df4d0736110bb637d7500ba8cb2432d6b0.

KaiWang-NOAA committed:
Revert "Merge branch 'develop' into CMAQv5.4_fixedRsnow"

This reverts commit 9ed2a9df4d0736110bb637d7500ba8cb2432d6b0, reversing
changes made to d8e0cf0f0f21b632938d4e8d7232dcd8f3c865f3.

KaiWang-NOAA commented:
Close the PR since the update is included in https://github.com/NOAA-EMC/AQM/pull/121
